### ISSUE TITLE
Symlinked source directory

### ISSUE THREAD

**Issue by joeyhoer** [SassDoc/sassdoc#372]
If the source directory is a symlink, `sassdoc` throws an error and will not compile:

```
> sassdoc symlink

events.js:72
        throw er; // Unhandled 'error' event
              ^
Error: EISDIR, read
```

**Comment by valeriangalliat** [comment]
Thanks for reporting.

Related: wearefractal/vinyl-fs#39.

**Comment by valeriangalliat** [comment]
Ok I don't think there's anything we can do in SassDoc for this.

Meanwhile, you can run `sassdoc symlink/**/*.scss` instead of `sassdoc symlink`.

**Comment by HugoGiraudel** [comment]
> Ok I don't think there's anything we can do in SassDoc for this.

We could add it in the docs.

**Comment by valeriangalliat** [comment]
Will be fixed with vinyl-fs 0.4.0, we'll just have to make sure to bump our requirement to `^0.4.0` when released.

**Comment by valeriangalliat** [comment]
Fixed with 2.1.2.
